framework/source/class/qx/event/handler/GestureCore.js

```diff
@@ -59,7 +59,31 @@ qx.Bootstrap.define("qx.event.handler.GestureCore", {
      * @type {Integer} Factor which is used for adapting the delta of the mouse wheel
      * event to the roll events,
      */
-    ROLL_FACTOR : 18
+    WHEEL_ROLL_FACTOR: 18,
```

Comment:
Theoretically, this is an API change. `qx.event.handler.GestureCore.ROLL_FACTOR` vs. `qx.event.handler.GestureCore.WHEEL_ROLL_FACTOR` although it is very unlikely that someone uses this outside of the framework.